py/src/test_call.py

```diff
@@ -153,15 +118,22 @@ def left_pad(b, to_length):
         assert len(b) <= to_length
         return b"\x00" * (to_length - len(b)) + b
 
-    cur.execute(
-        "insert into contract_code (hash, definition) values (?, ?)",
-        [
-            bytes.fromhex(
-                "050b2148c0d782914e0b12a1a32abe5e398930b7e914f82c65cb7afce0a0ab9b"
-            ),
-            compile_test_contract(),
-        ],
-    )
+    with open("crates/pathfinder/fixtures/contract_definition.json.zst", "rb") as file:
```

Comment:
was just thinking this should fail, and it did, ... need to make this file relative.